lib/mosaic-database/src/main/java/org/eclipse/mosaic/lib/database/persistence/SQLiteLoader.java

```diff
@@ -143,23 +147,25 @@ public void saveToFile(@Nonnull Database database, @Nonnull String fileName) {
                 Statement statement = connect();
                 log.debug("setting version and properties...");
                 saveProperties(database);
-                log.debug("Saving nodes...");
+                log.info("Saving {} nodes...", database.getNodes().size());
```

Comment:
Why do we assume INFO level here?
Because saving a db is happening less often?

Reply:
Yes. We only save the database when using scenario-convert and currently get no feedback except "Database saved". Now you additionally see how many objects have been stored., which gives you fast feedback when using scenario-convert. E.g. you immediately see how many routes are stored in the Database.

Another point is, that it's not possible to change the log-level when using scenario-convert, as the logback.xml is hidden in the jar file and no option to override the log level exists. We should change that somehow I guess.